src/System.Collections.NonGeneric/src/System/Collections/ArrayList.cs

```diff
@@ -101,6 +110,11 @@ public ArrayList(ICollection c)
         // the internal array used to hold items.  When set, the internal 
         // array of the list is reallocated to the given capacity.
         // 
+        /// <summary>Gets or sets the number of elements that the <see cref="T:System.Collections.ArrayList" /> can contain.</summary>
+        /// <returns>The number of elements that the <see cref="T:System.Collections.ArrayList" /> can contain.</returns>
+        /// <exception cref="T:System.ArgumentOutOfRangeException"><see cref="P:System.Collections.ArrayList.Capacity" /> is set to a value that is less than <see cref="P:System.Collections.ArrayList.Count" />.</exception>
+        /// <exception cref="T:System.OutOfMemoryException">There is not enough memory available on the system.</exception>
+        /// <filterpriority>1</filterpriority>
```

Comment:
I can also change the code to remove those. But that means IntelliSense won't have that either (not sure if it's necessary, I just know it's there today). I'm not familiar with this and how this was generated in the past (our current tool doesn't add those and that could be a bug)